content/codeofconduct.md

```diff
@@ -33,10 +33,8 @@ If you believe someone is violating the Code of Conduct, please report this in a
 
 You can make a personal or anonymous report through any of the following channels:
 
-Email: Send a message to any or all Diversity Chairs, who can be reached at oriol.abril.pla@gmail.com and andorra.alexandre@gmail.com. Event organizers will respond promptly.
-Slack: Join the PyMCon Slack and contact the Diversity Chairs, Oriol Abril-Pla or Alex Andorra.
-Online Form: A variant of the NumFOCUS Code of Conduct Report is available for PyMCon here (your name and contact information are not required).
-During PyMCon, we recommend contacting via email or Slack so we can take immediate action if necessary. In addition to contacting directly the Diversity committee, we also encourage you to make a report using our reporting form.
+Email: Send a message to pymconference@gmail.com address
```

Comment:
Will this email be dedicated to coc issues? Who will be reading the messages that get there? 

Some comments:
* only the code of conduct commitee should have access to coc reports, so if this email is used for other things I would not use it for that
* Already having the report form, if the response time of the email won't be significantly faster than the form then I am not sure it is worth it.